### ISSUE TITLE
Highcharts 4.2.6 SVG is breaking exporting

### ISSUE THREAD

**Issue by ShevaDas** [highcharts/highcharts#5590]
#### Expected behaviour
When exporting a graph to PNG or JPG using our personal PHP/Batik server, the graph should export.

#### Actual behaviour
No errors are thrown, but a completely empty image (no data) is returned.

#### Live demo with steps to reproduce
Something between 4.2.5 and 4.2.6 has changed such that graphs exported from 4.2.6 now contain SVG that causes Batik to silently fail when rasterizing the SVG, likely the same issue as described [here](https://issues.apache.org/jira/browse/BATIK-1116). I've attached a zip file containing the same graph, exported to SVG from both 4.2.5 and 4.2.6.

The line causing it to fail in 4.2.6 is: 
 ```<g class="highcharts-tooltip" style="cursor:default;padding:0;pointer-events:none;white-space:nowrap;" transform="translate(0,-9000000000)"```

Compared to 4.2.5, which works fine:
 ```<g class="highcharts-tooltip" style="cursor:default;padding:0;pointer-events:none;white-space:nowrap;" transform="translate(0,-9999)"```

[Highcharts exports.zip](https://github.com/highcharts/highcharts/files/420932/Highcharts.exports.zip)

#### Affected browser(s)
All

**Comment by TorsteinHonsi** [comment]
@oysteinmoseng Another case related to the tooltip hiding.

**Comment by oysteinmoseng** [comment]
Same cause as in #5570

**Comment by aadrian** [comment]
Any idea when 4.2.7 will be released? this export bug is pretty breaking :( .

Thank you.

**Comment by TorsteinHonsi** [comment]
We will start regression testing 4.2.7 next week, so you can expect a release in one or two weeks.

Until then, here's a drop-in fix for v4.2.6:
```js
// Workaround for #5590
Highcharts.wrap(Highcharts.Chart.prototype, 'sanitizeSVG', function (proceed, svg) {
	return proceed.call(this, svg)
    	.replace('-9000000000', '-9999');
});
```

[View live demo](http://jsfiddle.net/highcharts/u2v6t11v/).

**Comment by aadrian** [comment]
> Until then, here's a drop-in fix for v4.2.6 ...

Wow ! Thank you :) !
